.ci/eslint-plugin-zotero-translator/lib/translators.js

```diff
@@ -0,0 +1,213 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const jsonParse = require('jsonify').parse;
```

Comment:
Can't we just use `JSON.parse()`?

Reply:
We can, but we'd lose the position of the error.

Reply:
That's fine. I'd rather remove the dependency.

Reply:
Done